docs/v7_breaking_changes.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Taquito version 7
+author: Roxane Letourneau
+---
+
+:::warning Breaking changes
+With this major number update, we decided to refract some parts of the Taquito code. Please note that the migration to version 7 introduces breaking changes. 
+:::
+
+Each breaking change is exposed on this page with:
+- the reasons that motivated it,
+- code examples that show what needs to be changed on your code (how it was in prior versions versus how it needs to be using v7), and
+- potential errors you might get when updating to v7 without making the fixes.
+
+## Removed the default RPC Node URL
+
+Before version 7, Taquito was configured to use our default RPC node; the default value was set in the constructor of the `RpcClient` class. We took this approach so that users can get started quickly, and Taquito should "just work" with minimal fiddling. Users could import a ready-to-use `Tezos` singleton, an instance of the `TezosToolkit` class using the default RPC URL.    
+
+However, in version 7 of Taquito, we decided to remove the default RPC node URL. The reason behind this choice is to avoid that a lot of applications rely on our default node and thus centralize a part of the ecosystem on one node.
```

Comment:
"The reason behind this choice is to avoid that a lot of applications rely on our default node and thus centralize a part of the ecosystem on one node."

change to:

"The reason behind this choice is to encourage developers to make their own informed choice on which Tezos RPC node (public or private) is best for them. This change also helps avoid dApps using Taquito from centralizing on one public RPC node. Decentralization is an important part of Tezos, and we want to encourage decentralization at the RPC infrastructure level."